nnvm/tests/python/frontend/onnx/test_graph.py

```diff
@@ -18,8 +19,16 @@ def compare_graph(onnx_file, nnvm_sym, ishape):
     graph_util.check_graph_equal(g1, g2)
 
 def test_super_resolution_example():
-    fname, symbol = super_resolution, super_resolution_sym
+    fname, symbol = "super_resolution.onnx", super_resolution_sym
     compare_graph(fname, symbol, ishape=(1, 1, 224, 224))
 
+def test_squeeze_net():
+    # Only works for model downloaded from
+    # https://github.com/onnx/models/tree/master/squeezenet
+    fname = "squeezenet1_1.onnx"
```

Comment:
I tested with the model in model zoo first and got this error:
```
Traceback (most recent call last):
  File "/Users/hlu/github/tvm/nnvm/tests/python/frontend/onnx/test_graph.py", line 32, in <module>
    test_squeeze_net()
  File "/Users/hlu/github/tvm/nnvm/tests/python/frontend/onnx/test_graph.py", line 28, in test_squeeze_net
    compare_graph(fname, symbol, ishape=(1, 3, 224, 224))
  File "/Users/hlu/github/tvm/nnvm/tests/python/frontend/onnx/test_graph.py", line 19, in compare_graph
    graph_util.check_graph_equal(g1, g2)
  File "/Users/hlu/github/tvm/nnvm/python/nnvm/compiler/graph_util.py", line 94, in check_graph_equal
    raise ValueError("Graph compare error: " + err)
ValueError: Graph compare error: Number of nodes mismatch (170 v.s 118)
```
Then I tested with the model I downloaded from the onnx modelzoo (https://github.com/onnx/models/tree/master/squeezenet), the test passed. 
@zhreshold, do you mind to update the model in model_zoo? The graph is also different from the model in the onnx model zoo

Reply:
Can you have a quick debug why the old onnx model is no longer working? Just to make sure we don't break the legacy model support.